newspaper/views.py

```diff
@@ -195,6 +196,18 @@ def news():
     return render_template("news.html",
                            posts=get_news(choice="BBC"))  # TODO remove the hardcoded choice, make it a user preference
 
+@bp.route("/api/github/trending")
```

Comment:
Do we want an API route for this?
I think we should implement it in the newspaper that gets sent to the user.

Reply:
> Do we want an API route for this?
> I think we should implement it in the newspaper that gets sent to the user.

We probably don't need it, I just added it for debugging and it looks cool. We should definitely implement this in the newspaper